awx/settings/defaults.py

```diff
@@ -1148,13 +1148,8 @@
 
 # Settings for the ansible_base RBAC system
 
-# Only used internally, names of the managed RoleDefinitions to create
-ANSIBLE_BASE_ROLE_PRECREATE = {
-    'object_admin': '{cls.__name__} Admin',
-    'org_admin': 'Organization Admin',
-    'org_children': 'Organization {cls.__name__} Admin',
-    'special': '{cls.__name__} {action}',
-}
+# This has been moved to data migration code
+ANSIBLE_BASE_ROLE_PRECREATE = {}
```

Comment:
This was a big goof. Since I had put up https://github.com/ansible/django-ansible-base/pull/296 I confused myself about the contract with DAB. This was _unexpectedly_ creating the managed role definitions in the post_migrate hook, _in addition to_ AWX logic I added that did the same thing. Had it been removed in DAB (i.e. 296 was merged) that might have worked, but the path forward isn't super clear for that and I have cold feet on it.

In other words, we had duplicate role definitions which was causing unexpected names to be found.